nncf/quantization/algorithms/fast_bias_correction/algorithm.py

```diff
@@ -341,13 +323,13 @@ def get_statistic_points(self, model: TModel) -> StatisticPointsContainer:
         for node in biased_nodes:
             if not self._is_node_with_bias(node):
                 continue
-            input_tensor_names, _ = self._backend_entity.get_tensor_names(node)
-            edge_name = input_tensor_names[0]
+            input_port_id, output_port_id = 0, 0
```

Comment:
Is it always true for all backends and `layers_with_bias_types`?

Reply:
I don't think so. I can make a backend method `get_bias_input_output_port_ids(node: NNCFNode) -> Tuple[int, int]`
What do you think?

Reply:
Done